### PR TITLE
Remove leading '_' from all source file names

### PR DESCRIPTION
**Pull request by Page-Not-Found**
Remove leading '_' from all source file names

* The Class name is also renamed to match the file name.
* In places where there were multiple files with the same name within the same folder, a suffix of "internal" has been added based on the suggestion.

This PR is for #3116

### CODE REVIEW

File: src/System.Net.Security/src/System.Net.Security.csproj [hunk 1]
```diff
@@ -19,31 +20,28 @@
...
-
   <ItemGroup>
+    <Compile Include="System\Net\SecureChannel.cs" />
+    <Compile Include="System\Net\SecureProtocols\FixedSizeReader.cs" />
+    <Compile Include="System\Net\SecureProtocols\HelperAsyncResults.cs" />
+    <Compile Include="System\Net\SecureProtocols\SslState.cs" />
+    <Compile Include="System\Net\SecureProtocols\SslStreamInternal.cs" />
+    <Compile Include="System\Net\SecurityBuffer.cs" />
+    <Compile Include="System\Net\SecurityBufferType.cs" />
+    <Compile Include="System\Net\SslSessionsCache.cs" />
```
Comment (stephentoub) [inline src/System.Net.Security/src/System.Net.Security.csproj]: Why were all of these moved to the top?

File: src/System.Net.Sockets/System.Net.Sockets.sln [hunk 2]
```diff
@@ -57,10 +57,10 @@ Global
...
 		{43311AFB-D7C4-4E5A-B1DE-855407F90D1B}.Debug|Any CPU.ActiveCfg = Windows_Debug|Any CPU
 		{43311AFB-D7C4-4E5A-B1DE-855407F90D1B}.Debug|Any CPU.Build.0 = Windows_Debug|Any CPU
-		{43311AFB-D7C4-4E5A-B1DE-855407F90D1B}.FreeBSD_Debug|Any CPU.ActiveCfg = FreeBSD_Debug|Any CPU
-		{43311AFB-D7C4-4E5A-B1DE-855407F90D1B}.FreeBSD_Debug|Any CPU.Build.0 = FreeBSD_Debug|Any CPU
-		{43311AFB-D7C4-4E5A-B1DE-855407F90D1B}.FreeBSD_Release|Any CPU.ActiveCfg = FreeBSD_Release|Any CPU
-		{43311AFB-D7C4-4E5A-B1DE-855407F90D1B}.FreeBSD_Release|Any CPU.Build.0 = FreeBSD_Release|Any CPU
+		{43311AFB-D7C4-4E5A-B1DE-855407F90D1B}.FreeBSD_Debug|Any CPU.ActiveCfg = Windows_Release|Any CPU
+		{43311AFB-D7C4-4E5A-B1DE-855407F90D1B}.FreeBSD_Debug|Any CPU.Build.0 = Windows_Release|Any CPU
+		{43311AFB-D7C4-4E5A-B1DE-855407F90D1B}.FreeBSD_Release|Any CPU.ActiveCfg = Windows_Release|Any CPU
+		{43311AFB-D7C4-4E5A-B1DE-855407F90D1B}.FreeBSD_Release|Any CPU.Build.0 = Windows_Release|Any CPU
```
Comment (stephentoub) [inline src/System.Net.Sockets/System.Net.Sockets.sln]: This change likely happened automatically due to the .csproj not having a FreeBSD configuration.  It's strange to have the FreeBSD configuration in the solution end up being the release builds.  We should probably either remove the FreeBSD config until there's an implementation for it, or add such a config to the .csproj and leave this as it was before.

**Comment by stephentoub** [comment]
Thanks for the cleanup, @Page-Not-Found.

cc: @cipop, @davidsh for the networking changes.
@weshaggard, FYI.

File: src/System.Net.Security/src/System.Net.Security.csproj (hunk 1 above)
Comment (CIPop) [inline src/System.Net.Security/src/System.Net.Security.csproj]: +1 Please keep the files sorted in lexicographical order.

File: src/System.Net.Security/src/System.Net.Security.csproj [hunk 3]
```diff
@@ -60,8 +58,7 @@
     <Compile Include="$(CommonPath)\System\Net\InternalException.cs">
       <Link>Common\System\Net\InternalException.cs</Link>
     </Compile>
-
-    <!-- Debug only --> 
+    <!-- Debug only -->
```
Comment (CIPop) [inline src/System.Net.Security/src/System.Net.Security.csproj]: Nit: We edit csproj files manually and so I prefer having these spaces.

**Comment by CIPop** [comment]
:+1: for NCL code with a few nitpicks regarding csproj formatting.

File: src/System.Net.Security/src/System.Net.Security.csproj (hunk 1 above)
Comment (Page-Not-Found) [inline src/System.Net.Security/src/System.Net.Security.csproj]: I will rearrange files as previous like and just remove the "_" form them. Is that okie to do it?

File: src/System.Net.Security/src/System.Net.Security.csproj (hunk 3 above)
Comment (Page-Not-Found) [inline src/System.Net.Security/src/System.Net.Security.csproj]: I will get this changed in the next PR

File: src/System.Net.Sockets/System.Net.Sockets.sln (hunk 2 above)
Comment (Page-Not-Found) [inline src/System.Net.Sockets/System.Net.Sockets.sln]: I am not sure what to do with this? Do I just revert these changes from the commit.

Comment (davidsh) [inline src/System.Net.Sockets/System.Net.Sockets.sln]: >or add such a config to the .csproj and leave this as it was before.

We should do this.  Simply fix it up and push a new commit to the PR.

**Comment by stephentoub** [comment]
Thanks, @Page-Not-Found.
